src/bridging-script/ios-bridge.js

```diff
@@ -0,0 +1,29 @@
+let MiniAppBridge = {}
```

Comment:
Looks good. 
@ts-leojoseph-r Can we add the generic one which works for the platform?

Reply:
Sorry, I don't get it. Platform can access it using `window.MiniApp.getUniqueId()` which is generic. We created MiniAppBridge object to move the exec and execCallback methods inside an object.

Reply:
Maybe this should be defined as `window.MiniAppBridge`?

Reply:
Regarding having generic JS for both platforms: we had discussed having three files, something like `common-bridge.js`, `ios-bridge.js`, and `android-bridge.js`. However, this will require us to bundle the JS before it can be used on the native side. That's a little tricky because then we will need to have the bundled JS committed to the repo.

I guess we could setup the CI job so that it will commit the bundled JS every time something is merged to master. That would also allow us to use TypeScript and ensure that the bundled JS is targeting ES5.

Here's my suggestion:
1. For now, keep what we have here and don't use bundling.
2. Rename this file to just `bridge.js`, and then for the Android implementation we can add platform detection for knowing whether to use `MiniApp.postMessage` or `webkit.messageHandlers.MiniApp.postMessage`.
3. In the future, we can add bundling, TypeScript, and split into separate files. Also will setup CI to commit the bundled JS when merging to master.

This will allow us to be unblocked on the native side for the moment. What do you think?

Reply:
We can also inject the JS which was hosted remotely. If we get to bundle the generic JS and host it and we can use the same in the native.
But for now, I agree with the suggestions and I will make changes accordingly

Reply:
> Maybe this should be defined as `window.MiniAppBridge`?
I don't feel good about attaching the `MiniAppBridge` object to the window variable. As we are already having `window.MiniApp` in window object. What do you think ?